netbout-web/src/main/java/com/netbout/rest/account/TkSaveEmail.java

```diff
@@ -144,4 +140,23 @@ public Response act(final Request req) throws IOException {
         }
         return res;
     }
+
+    /**
+     * Combine verified email and received email into a single String with
+     * specific format that we use later.
+     * @param alias Alias
+     * @param email Received email
+     * @return Verified and received email properly combined together.
+     * @throws IOException If something wrong
+     */
+    private static String combinedEmails(final Alias alias, final String email)
+        throws IOException {
+        final String verified;
+        if (alias.email().contains("!")) {
```

Comment:
@CarlosGines Let's extract `alias.email()` into a final variable to improve readibility.